data_steward/tools/deid_runner.sh

```diff
@@ -50,16 +55,12 @@ gcloud auth activate-service-account --key-file="${key_file}"
 gcloud config set project "${APP_ID}"
 
 cdr_deid="${cdr_id}_deid"
-
-#------Create de-id virtual environment----------
-set -e
-
-# create a new environment in directory deid_env
 ROOT_DIR=$(git rev-parse --show-toplevel)
 DATA_STEWARD_DIR="${ROOT_DIR}/data_steward"
 TOOLS_DIR="${DATA_STEWARD_DIR}/tools"
-DEID_DIR="${DATA_STEWARD_DIR}/deid" 
+DEID_DIR="${DATA_STEWARD_DIR}/deid"
 
+#------Create de-id virtual environment----------
 virtualenv -p "$(which python3.7)" "${DATA_STEWARD_DIR}/curation_venv"
 
 source "${DATA_STEWARD_DIR}/curation_venv/bin/activate"
```

Comment:
This skips the part about deid making a dataset and filling it with the data.  If I remember correctly, it's done programmatically via python.  That may have to be an extra jira issue.

Reply:
It is still being used in the script after setting up the environment.

Reply:
I had to look back over the diagram and the deid code.  I was worried about it, but now I don't think it's an issue.